code/datums/skills.dm

```diff
@@ -158,10 +178,10 @@ engineer, construction, leadership, medical, surgery, pilot, police, powerloader
 #undef SKILLSID
 
 /datum/skills/proc/getPercent(rating, max_rating)
-	return CLAMP01(vars[rating] * 100 / max_rating * 0.01)
+	return CLAMP01(max(vars[rating], 0) * 100 / max_rating * 0.01)
 
 /datum/skills/proc/getRating(rating)
-	return vars[rating]
+	return max(vars[rating], 0)
```

Comment:
```suggestion
	return vars[rating]
```
Turns out negative skill ratings exist and are relied on for a couple things already, smartgun penalties being the most prominent.